eclair-core/src/main/scala/fr/acinq/eclair/balance/BalanceActor.scala

```diff
@@ -85,9 +85,20 @@ private class BalanceActor(context: ActorContext[Command],
       res match {
         case Success(result) =>
           log.info("current balance: total={} onchain.confirmed={} onchain.unconfirmed={} offchain={}", result.total.toDouble, result.onChain.confirmed.toDouble, result.onChain.unconfirmed.toDouble, result.offChain.total.toDouble)
-          log.debug("current balance details : {}", result)
-          if (result.onChain.confirmed < 1.millibtc) {
-            context.system.eventStream ! EventStream.Publish(NotifyNodeOperator(NotificationsLogger.Warning, s"on-chain confirmed balance is low (${result.onChain.confirmed.toMilliBtc}), eclair may not be able to guarantee funds safety in case channels force-close: you should add some utxos to your bitcoin wallet"))
+          log.debug("current balance details: {}", result)
+          // This is a very rough estimation of the fee we would need to pay for a force-close with 5 pending HTLCs at 100 sat/byte.
+          val perChannelFeeBumpingReserve = 0.5.millibtc
+          // Instead of scaling this linearly with the number of channels we have, we use sqrt(channelsCount) to reflect
+          // the fact that if you have channels with many peers, only a subset of these peers will likely be malicious.
+          val estimatedFeeBumpingReserve = perChannelFeeBumpingReserve * Math.sqrt(result.channelsCount)
+          if (result.onChain.confirmed < estimatedFeeBumpingReserve) {
```

Comment:
We should consider the _safe_ amount instead, which includes unconfirmed change.

Reply:
I'm not sure we can, because BIP125 has restrictions on unconfirmed inputs, even if they're safe, so there's a chance that we actually can't use these inputs for RBF...so I'd rather be very conservative here and consider only confirmed utxos.